go/chat/boxer.go

```diff
@@ -219,6 +222,10 @@ func (b *Boxer) boxMessageV1(ctx context.Context, msg chat1.MessagePlaintextV1,
 		}
 	}
 
+	if len(msg.ClientHeader.TlfName) == 0 {
```

Comment:
OK; will keep that in mind for future. Does that apply to client<->service and kbfs<->service too, considering they are always shipped together?